### DISCUSSION TITLE
Browser extension: able to report a website that doesn't accept a SimpleLogin domain

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#681] [Feature Requests]
Some websites (mistakenly) don't accept some SimpleLogin domains. They often use some email checking service that don't like email alias or even worse, only accept popular email services like Gmail, Outlook, etc.

Having a way to report these websites could 
- alert other people when they visit them. 
- allow SimpleLogin community to reach out to them to address the issue.

**Comment by WhamboMPS** [comment]
FYI, sometimes the issue is that a website's form validation code improperly rejects any email address that includes a subdomain (e.g. `me@mydomain.com` would be accepted while `me@subdomain.mydomain.com` would fail). This is technically a bug on the third-party website, not anything inherently amiss with SL).

If the third-party user form fails entry of an email address that uses a subdomain there is a workaround. That is to create a new SL Alias based on one of SL's available general use aliases (e.g. `simplelogin.co`, `aleeas.com`, or `8shield.net`). These don't use subdomains and so should be accepted on the third-party form (the period in the email address prefix for these domains should not cause a problem).

It is, of course, hypothetically possible for third-party sites to block registrations from specific domains. Under such a scenario blocking of SL-owned domains _could_ be occurring. However, given SL's scale I doubt this happening and I would suggest that this issue is such a corner case that this enhancement should be prioritized quite low, if indeed it should be completed at all.

I have used subdomain-based aliases for many years and so have occasionally encountered this issue. Shown below are examples of domains where a subdomain-based email address fails but an-SL owned domain that does not use a subdomain works. These examples should add credence to the argument that this is not an SL issue, it is a third-party form validation code issue:

`hallmark.com`
`nytimes.com`
`quicken.com`
`schwab.com`

**Reply by nguyenkims** [reply]
Thanks for the info on subdomains, that's interesting to know blocking subdomains is actually a common practice. 

> It is, of course, hypothetically possible for third-party sites to block registrations from specific domains. Under such a scenario blocking of SL-owned domains could be occurring. 

Unfortunately this can happen as websites use a shared list of disposable domains. There was an attempt to add SimpleLogin to such list but it has been stopped: https://github.com/ivolo/disposable-email-domains/issues/846
